test/py/explicit_module/runtest.sh

```diff
@@ -25,10 +25,12 @@ rm -rf gen-py
 ../../../compiler/cpp/thrift --gen py test3.thrift && exit 1  # Fail since test3.thrift has python keywords
 ../../../compiler/cpp/thrift --gen py:enum shared_types.thrift || exit 1
 ../../../compiler/cpp/thrift --gen py:enum test4.thrift || exit 1
+../../../compiler/cpp/thrift --gen py:enum test5.thrift || exit 1
```

Comment:
can you please also generate the code with both enum and slots (as slots also change the code significantly), you can use `--out` to put it under a different directory than `gen-py` so you can have another test around line 33 to also make sure that the combination of enum+slots still works.

Reply:
I am not sure what slots are and how I can generate code that has enum and slots. Is there an example how I can generate code with enum and slots? I tried variations of `py:enum py:slots` and such but no such luck.

Reply:
NVM I figured it out and it seems to work. Is there a specific test case you have in mind? I just changed this - `../../../compiler/cpp/thrift --gen py:enum,slots test5.thrift` and the new serialization/deserialization test  that I added works.

Reply:
yea can you add that into the PR so we make sure neither `enum` and `enum,slots` will break in the future?

Reply:
Added the slots option to the existing test5 test!